src/Illuminate/Cache/Repository.php

```diff
@@ -115,8 +115,7 @@ public function get($key, $default = null)
      *
      * Items not found in the cache will use the provided default value, or `null` if not specified.
      *
-     * @param  array  $keys A list of keys that can obtained in a single operation. To specify a default value, use the
```

Comment:
What was wrong with the original documentation. It more accurately represents what the function does.

Reply:
We don't document parameters individually anywhere in our open source projects.